### DISCUSSION TITLE
How to find total liquid asset of my account?

### DISCUSSION THREAD

**Discussion by wenxie18** [blankly-finance/blankly#180] [Q&A]
Hi, I want to check the total asset (cash + current values of all purchased assets) in my account. Does anyone know if blankly has a  direct command? I only found state.interface.cash / state.interface.account['USD']. Thanks.

**Comment by EmersonDove** [comment]
Hey we don't have a native way of valuing your account, but you can easily loop through all your nonzero assets using `interface.get_account()` and then multiplying that with `interface..get_price()`
